data-reconciliation-app/src/auth/Jwt-validation.ts

```diff
@@ -0,0 +1,42 @@
+import * as ccfapp from "@microsoft/ccf-app";
+import { ServiceResult } from "../utils/service-result";
+import msIdProvider from "./jwtIssuer/MS-AAD";
+import testIdProvider from "./jwtIssuer/TestIdP";
+
+/**
+ * JWT Identity Providers
+*/
+export enum JwtIdentityProviderEnum {
+    MS_AAD = "https://login.microsoftonline.com/common/v2.0",
+    Test = "https://demo"
```

Comment:
Maybe we call it `Demo` then?

Reply:
Will change it to "https://test" instead.

Reply:
have changed it to demo.